duplex-messaging.md

```diff
@@ -0,0 +1,74 @@
+# SMP agent protocol - duplex communication over SMP protocol
+
+## Table of contents
+
+- [Abstract](#abstract)
+- [SMP agent commands](#smp-agent-commands)
+
+## Abstract
+
+Protocol describes the command that can be sent to the client that communicates with multiple SMP servers. This protocol allows to manage multiple simplex connections organised into groups defining duplex communication channels.
```

Comment:
adjust wording here maybe? it's a bit hard to understand right now, stress that client is the **SMP agent** maybe and also that the one who sending is the user (like below on line 16 - the user and SMP agent), as well as that communication goes both ways, albeit with different commands?
e.g.
> Protocol describes commands by which the user and SMP agent (that communicates with multiple SMP servers) communicate between each other.

also "organi**z**ed" for consistency